contracts/truefi2/strategies/CurveYearnStrategy.sol

```diff
@@ -28,8 +28,6 @@ contract CurveYearnStrategy is UpgradeableClaimable, ITrueStrategy {
 
     // Number of tokens in Curve yPool
     uint8 public constant N_TOKENS = 4;
-    // Max slippage during the swap
-    uint256 public constant MAX_PRICE_SLIPPAGE = 200; // 2%
```

Comment:
Are we sure that we can remove constant from the memory? @vanruch?

Reply:
Nevermind.

Reply:
My understanding is that constants do not affect contract storage. See the Solidity [docs](https://docs.soliditylang.org/en/latest/contracts.html#constants) for reference:

> It is also possible to define constant variables at the file level.
>
> The compiler does not reserve a storage slot for these variables, and every occurrence is replaced by the respective value.